browser/main/lib/RcParser.js

```diff
@@ -0,0 +1,23 @@
+import _ from 'lodash'
+
+const path = require('path')
+const sander = require('sander')
+
+function parse (boostnotercPath) {
+  if (!sander.existsSync(boostnotercPath)) return
```

Comment:
Maybe `return {};` is better because return value is used for `Object.prototype.assign`.